34064: revert-layer (#34085)
mdn/content: files/en-us/web/css/revert-layer/index.md

> The `revert-layer` keyword lets you rollback styles to the ones specified in previous cascade layers. All cascade layers exist in the [author origin](/en-US/docs/Glossary/Style_origin). The {{cssxref("revert")}} keyword, in comparison, lets you remove styles applied in the author origin and roll back to styles in user origin or user-agent origin.

Reviewer: I think we can improve this as well
```suggestion
The `revert-layer` keyword lets you rollback styles to those specified in previous cascade layers within the [author origin](/en-US/docs/Glossary/Style_origin). The {{cssxref("revert")}} keyword, in comparison, lets you roll back styles applied in the author origin to those specified in the user origin or user-agent origin.
```

> The `revert-layer` keyword is ideally meant for applying on properties inside a layer. However, if the `revert-layer` keyword is set on a property outside a layer, the value of the property will roll back to any value set by presentational hints (e.g. the `width` or `height` attributes in HTML), defaulting to the value established by the user agent's stylesheet (or by user styles, if any exist). So in this scenario, the `revert-layer` keyword behaves very similarly to the The `revert-layer` keyword is ideally meant for applying on properties inside a layer. However, if the `revert-layer` keyword is set on a property outside a layer, the value of the property will roll back to the default value established by the user agent's stylesheet (or by user styles, if any exist). So in this scenario, the `revert-layer` keyword behaves like the {{cssxref("revert")}} keyword, with the exception that `revert` considers presentational hints to be part of the author origin, so it reverts them too.

Reviewer: - Some bits in the original text can be refined (I think I wrote this page originally ;) ). Made the previous wording a bit more compact.
- Suggesting rewording to highlight the difference between `revert-layer` and `revert` instead of using "behaves similarly" at all.
- There's some duplicate text, probably copy paste error
```suggestion
The `revert-layer` keyword is ideally meant to be applied to properties within a cascade layer. However, if applied to properties outside a cascade layer, it rolls back property values to any values set by presentational hints (such as `width` and `height` attributes or the `<s>` element in HTML), defaulting to the values established by the user agent's stylesheet or user styles. Unlike the `revert` keyword, which considers presentational hints as part of the author origin and reverts them as well, the `revert-layer` keyword ignores presentational hints outside the cascade layer, so it does not revert them.
```
